arbitrum-docs/stylus/stylus-quickstart.mdx

```diff
@@ -204,7 +204,7 @@ deployment tx total cost: "0.000712373700000000" ETH
 
 ### Deployment
 
-Let's move on to the contract's actual deployment. Two transactions will be sent onchain: the contract deployment and its [activation](/stylus/stylus-gentle-introduction.md#activation).
+Let's move on to the contract's actual deployment. Two transactions will be sent onchain: the contract deployment and its activation.
```

Comment:
This change should probably be in a different PR, not in this one.

Reply:
this is my fault, didn't switch branch, will revert